Sources/Build/Command.compile(SwiftModule).swift

```diff
@@ -20,36 +20,23 @@ extension Command {
             return Command(node: module.targetName, tool: tool)
         }
 
+        var args = ["-j8", "-D", "SWIFT_PACKAGE"]
```

Comment:
Let's use the same `numThreads` constant for this, so it isn't hard coded twice.